sdk/servicebus/service-bus/src/core/receiverHelper.ts

```diff
@@ -11,37 +13,61 @@ import { receiverLogger as logger } from "../log";
  * @internal
  */
 export class ReceiverHelper {
-  private _isSuspended: boolean = false;
+  private _isSuspended: boolean = true;
```

Comment:
What was the reasoning for changing this to `true` at initialization?  That seems a bit counter to a user initializing the suspend.

Reply:
The basic idea is - should we consider "suspended" or not if the user hasn't actively "resumed" it. 

Before it used to just not care but I made it be more explicit. So now it starts "off" by default, rather than "on".